core/pom.xml

```diff
@@ -160,5 +160,19 @@
             <version>1.5.2 </version>
             <scope>test</scope>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/org.apache.poi/poi -->
+		<dependency>
```

Comment:
can you change the tabs to spaces so the indentation aligns with the other dependencies